src/index.html

```diff
@@ -6,9 +6,53 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo">
+        <img
+        src="images/logo.png"
+        class="logo"
+        alt="Moyo Logo"
```

Comment:
Don't forget to check the checklist.
![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/385c9d12-0a34-4a19-8add-255224907fd8)